### PR TITLE
Include Protocol Conformance

### PR DESCRIPTION
**Pull request by jch-duran**

### CODE REVIEW

File: swift-style-guide.md [hunk 1]
```diff
@@ -219,3 +219,56 @@ _Rationale:_ Omitting redundant type parameters clarifies the intent, and makes
 
 #### Curly braces go on the same line
 In every situation.
+
+## Code Organization
+
+Use extensions to organize your code into logical blocks of functionality. Each extension should be set off with a `// MARK: -` comment to keep things well-organized.
```
Comment (swilliams) [inline swift-style-guide.md]: Good call.

Comment (swilliams) [inline swift-style-guide.md]: Add a blurb about how these marks show up in the 'show document items' menu (accessible via `⌃6`)

**Comment by swilliams** [comment]
@jch-duran Ping!

**Comment by jch-duran** [comment]
@swilliams There's still some clean up I'm performing, but I have the blurb about the `Show Document Items` included.

**Pull request merged**